src/main/resources/data/tetra/modules/bow/basic_string.json

```diff
@@ -12,6 +12,7 @@
         "tetra:enchantment/ma-enchants_floating",
         "tetra:enchantment/ma-enchants_paralysis",
         "tetra:enchantment/apotheosis_life_mending",
+		"tetra:enchantment/apotheosis_true_infinity",
```

Comment:
Could you change this to have the same indentation as the rest of the items in the array?